foundry/applications/formApplications/baseEntitySheets/journalSheet.d.ts

```diff
@@ -1,23 +1,67 @@
 /**
  * The JournalEntry Configuration Sheet
- *
- * @param entity - The JournalEntry instance which is being edited
```

Comment:
Could you add these params (fixed, i.e. `entity` replaced by `object`) to the constructor's TSDoc?